test/runner.js

```diff
@@ -30,6 +31,7 @@ if ( program.grep ) {
 mocha.suite.beforeAll( function() {
 	Chai.use( immutableChai );
 	Chai.use( sinonChai );
+	sinon.assert.expose( Chai.assert, { prefix: '' } );
```

Comment:
Yeah, this was mostly a stopgap to avoid having to do too much surgery on existing tests. I'd started to move over to the expect syntax and then chickened out. :chicken: